### ISSUE TITLE
Documentation about the bungeecord module + other questions

### ISSUE THREAD

**Issue by sgdc3** [BuycraftPlugin/BuycraftX#9]
I can't find any documentation about the new Bungeecord module... Can someone help me please? ;)

**Comment by minecrafter** [comment]
The BungeeCord plugin is meant to allow Buycraft to execute commands on your BungeeCord proxy. It does not enable any other special functionality

**Comment by sgdc3** [comment]
Ok, thanks ;) What about a BungeePerm hook in the future?

**Comment by minecrafter** [comment]
We are not going to add in special integrations. BungeePerms should have commands that the console can execute (BuycraftX runs each command as the console).

**Comment by sgdc3** [comment]
Right, thanks for the infos ;)

**Comment by sgdc3** [comment]
Wait, one last thing: the commands are called immediately or only when the player joins the server?

**Comment by minecrafter** [comment]
It depends on the settings for the command.

If the command requires the player be online, it will be run after they come online. If they do not, they will be run as soon as possible.

**Comment by sgdc3** [comment]
Ok, so an Authme/xAuth integration would be useful, right? If the owner wants to run the command only after a successful login the plugin should first check the login status via API.

**Comment by minecrafter** [comment]
As I have stated earlier, we are not going to add in special integrations for every plugin out there. I am now going to close this issue. If you have further questions, I invite you either to open a new issue or contact [support](https://www.buycraft.net/support).

**Comment by sgdc3** [comment]
Ok, thanks ;)
